save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/CosvSchemaUtils.kt

```diff
@@ -96,6 +94,22 @@ fun Float.asSeverity(): Severity = Severity(
     scoreNum = toString(),
 )
 
+/**
+ * @return list of VulnerabilityProjectDto
+ */
+@Suppress("TOO_MANY_LINES_IN_LAMBDA")
+fun CosvSchema<*, *, *, *>.getVulnerabilityProjects(): List<VulnerabilityProjectDto> = affected?.map {
```

Comment:
can we use `it.package` instead of VulnerabilityProjectDto?

Reply:
But we need not only the `package`, but also `versions` and `ranges`
And to create the COSV we use VulnerabilityDto on CreateVulnerabilityView, inside of which there is a list of VulnerabilityProjectDto

Reply:
> But we need not only the package, but also versions and ranges

Ok, can we use `affected` instead of `VulnerabilityProjectDto`?

> And to create the COSV we use VulnerabilityDto on CreateVulnerabilityView, inside of which there is a list of VulnerabilityProjectDto

It'll be removed. We can remove support of `VulnerabilityProjectDto` on creating COSV now

Reply:
Ok. Fixed it